ee/docker-ee/ubuntu.md

```diff
@@ -156,9 +156,11 @@ and update Docker Engine - Enterprise from the repository.
 6.  Use the following command to set up the **stable** repository. Use the
     command as-is. It works because of the variable you set earlier.
 
-    > **Note:** The `lsb_release -cs` sub-command below returns the name of your
+    > **Note** 
+    > 
+    > The `lsb_release -cs` sub-command below returns the name of your
     > Ubuntu distribution, such as `xenial`.
```

Comment:
In my opinion, the wording should be slightly changed "The following `lsb_release l-cs` sub command returns the name of your Ubuntu distribution (e.g., `xenial`).

Also, the command that follows should be a part of the note block.